sources/metadata/pmm-server/3.1.0.yaml

```diff
@@ -0,0 +1,7 @@
+version: 3.1.0
+imageInfo:
+  image_path: "percona/pmm-server:3.1.0"
```

Comment:
let's for now use 3-dev-latest and 3-dev-container

Reply:
so we can test actual update

Reply:
I think there might be a problem with this though:
* Our version parser cannot parse versions like `3-dev-container`, so we will need to create new tags anyway (e.g., for `3.0.0-dev-container`).
* We already don't consider versions with `Rest` part when checking version changelogs (https://github.com/percona/pmm/blob/5561f584c01297523d401b087c6c701c675ed3b2/managed/services/server/updater.go#L338)
  This is because we don't expect such versions to have a related release notes, and it wasn't clear what upgrading between them will look like.

Reply:
version still can be 3.1.0, but image can be changed to `perconalab/pmm-server:3-dev-container`

Reply:
Yeah that makes sense